src/main/java/com/zontik/groshiky/controllers/IndexController.java

```diff
@@ -30,10 +30,12 @@ public String index(ModelMap model) {
     }
 
     @RequestMapping(value = "/index", method = RequestMethod.GET)
-    public String index(@RequestParam(value = "error", required = false) String error, Model model) {
+    public String index(@RequestParam(value = "error", required = false) String error,
+                        @RequestParam(value = "success", required = false) String success,Model model) {
```

Comment:
I don't like the idea to add a new parameter for every possible operation result. Can we replace these two params with one enum?

Reply:
We need two parameters, but one of them provides spring. 
I do not know how to change it yet

Reply:
You can use failureUrl method to provide your own url with parameters in case of failure authentication